administrators.rst

```diff
@@ -287,27 +287,28 @@ search criteria. For further details of the API parameters, see the
 DuraCloud
 +++++++++
 
-.. seealso::
-
-   :ref:`Archivematica-DuraCloud Quick Start Guide
-   <archivematica:duracloud-setup>`
-
 Archivematica can use DuraCloud as an access protocol for the Storage Service in
-version 0.5 and higher. Typically, a Storage Service space has a one-to-one
-relationship with a space within DuraCloud.
+version 0.5 and higher. A Storage Service space has a one-to-one relationship
+to a space within DuraCloud.
+
+To set up your Archivematica instance with DuraCloud, please see :ref:`Using
+DuraCloud with Archivematica <archivematica:duracloud-setup>`.
 
 Fields:
 
-* **Size**: the maximum size allowed for this space. Set to 0 or leave blank
-  for unlimited. This field is optional.
-* **Path**: the absolute path to the Space on the local filesystem.
-* **Staging path**: the absolute path to a staging area. Must be UNIX filesystem
-  compatible and preferably will be located on the same filesystem as the path.
-* **Host**: the hostname of the DuraCloud instance, e.g.
-  ``example.duracloud.org``.
-* **User**: a username with sufficient permissions to permit authentication.
-* **Password**: the password of the above user.
-* **Duraspace**: the name of the Space within DuraCloud
+* **Access protocol**: DuraCloud
+* **Size**: *This field is optional.*
+* **Path**: Leave this field blank.
+* **Staging path**: A location on your local disk where Archivematica can
```

Comment:
There's an issue about the leading slash in the path and location's path,but is that true for the staging path? I thought that was supposed to be the absolute path on the local dish and include the leading slash.

Reply:
I thought it was about the location's relative path but now I'm not sure! The location's relative path is where you enter the DuraCloud space name, so therefore it's described in the related [AM docs PR](https://github.com/artefactual/archivematica-docs/pull/343) (see lines 81 and 94 in the diff.) Maybe @ross-spencer can confirm.